lib/onlyoffice_webdriver_wrapper/helpers/headless_helper/process_helper.rb

```diff
@@ -5,7 +5,8 @@ module ProcessHelper
     # @return [Nothing] kill specified process
     def kill_process(process)
       OnlyofficeLoggerHelper.log("killall -9 #{process} 2>&1")
-      `killall -9 #{process} 2>&1`
+      kill_result = `killall -9 #{process} 2>&1`
+      OnlyofficeLoggerHelper.log("killall -9 #{process}: #{kill_result}")
```

Comment:
OnlyofficeLoggerHelper.log("kill_process(#{process}) result: #{kill_result}") is better if we add some more options to kill command